content/05-expression.md

```diff
@@ -492,6 +492,43 @@ This syntax is also used to access types within packages in the form of `pack.Ty
 
 The typer ensures that an accessed field actually exist and may apply transformations depending on the nature of the field. If a field access is ambiguous, understanding the [resolution order](type-system-resolution-order) may help.
 
+##### since Haxe 4.3.0
+
+Null-safe field access is expressed using the `?.` operator. Such access checks if the object is not `null` first, and only then accesses the given field. If the object *is* `null`, then the entire expression has the value `null`.
+
+```haxe
+object?.fieldName
+```
+
+This is the same as:
+
+```haxe
+(object != null ? object.fieldName : null)
+```
+
+The null-safe field access can be chained, allowing a much cleaner access to deeply nested data where each step may be `null`, which may be the case when dealing with externs in a dynamically typed language, for example.
+
+
+
+<!--label:expression-null-coalesce-->
+### Null coalescing
+
+##### since Haxe 4.3.0
+
+Complementing [null-safe field access](expression-field-access), the null coalescing operator `??` allows specifying a default/fallback value if a given expression is `null`.
+
+```haxe
+value ?? "fallback"
+```
+
+This roughly the same as:
```

Comment:
Ah and here we go for the "roughly", which is more accurate. I think we can do that for `?.` too and also add the "not evaluated twice" clause.